orchagent/main.cpp

```diff
@@ -81,6 +81,8 @@ int32_t gVoqMaxCores = 0;
 uint32_t gCfgSystemPorts = 0;
 string gMyHostName = "";
 string gMyAsicName = "";
+uint64_t gSyncdRespTimeoutLC = ( 5 * SAI_REDIS_DEFAULT_SYNC_OPERATION_RESPONSE_TIMEOUT);
```

Comment:
do you need this variable? Can #define useful. these variables aren't used other than switchcreate.

Reply:
Fixed